db/seeds.rb

```diff
@@ -1,7 +1,7 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.create(
```

Comment:
Use a factory to create a seed for the user

Reply:
En este punto del training no habíamos configurado factory-bot ni faker, está bien si lo cambio en una rama que ya haya incluido las gemas ?